README.md

```diff
@@ -1,19 +1,35 @@
 ![GeoJS](http://opengeoscience.github.io/geojs/images/logo_256.png)
 [![Build Status](https://travis-ci.org/OpenGeoscience/geojs.svg?branch=master)](https://travis-ci.org/OpenGeoscience/geojs) [![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.15459.svg)](http://dx.doi.org/10.5281/zenodo.15459) ![repo size](https://reposs.herokuapp.com/?path=OpenGeoscience/GeoJS)
 =======
-Primarily developed at Kitware, GeoJS is a high performance visualization and analysis library targeted for GeoInformatics.
+Primarily developed by [Kitware](http://www.kitware.com) and
+[Epidemico](http://epidemico.com/), GeoJS is a high performance
+visualization and analysis library targeted for GeoInformatics.
 
-![Historical hurricane data Visualization in GeoJS](http://opengeoscience.github.io/geojs/examples/hurricanes/thumb.jpg "")
+![Historical hurricane data Visualization in GeoJS]
+(http://opengeoscience.github.io/geojs/examples/hurricanes/thumb.jpg "")
 
 Documentation
 ==============
-You can find more information on GeoJS installation and usage in our [Documentation](http://geojs.readthedocs.org/en/latest/index.html). For live examples visit [Github.io](http://opengeoscience.github.io/geojs/examples/index.html).
+You can find more information on GeoJS installation and usage in our [Documentation](http://geojs.readthedocs.org/en/latest/index.html).
+For live examples visit [Github.io](http://opengeoscience.github.io/geojs/examples/index.html).
+
+Support
+==============
+We would love to get your feedback and bug reports. Please send us an email at
+[geojs-developers](http://public.kitware.com/mailman/listinfo/geojs-developers)
+for questions related to feature requests, roadmap or in-depth
+technical discussions. Use [geojs-users](http://public.kitware.com/mailman/listinfo/geojs-users)
+if you have general questions or bug reports on GeoJS.
 
 License
 ============
-Copyright 2015 Kitware Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
```

Comment:
@jbeezley thanks...for catching this. I will update the license file and the license file in package.json.